### DISCUSSION TITLE
Bagaimana cara Deploy app Flask Dashboard

### DISCUSSION THREAD

**Discussion by ammaralwandi97** [teamalgoritma/community#836] [Q&A] [answered]
Bagaimana cara Deploy Flask Dashboard app hasil Capstone Xion DA ke internet?
Menurut saya sayang kalau masih di lokal, mau saya cantumkan di portofolio saya 😄

**Comment by finesaaa** [comment, marked as answer]
Dear pak @ammaralwandi97 
Silahkan boleh diikuti step-step deploy Flask dari dokumentasi resminya disini, semoga membantu pak
https://docs.render.com/deploy-flask
